mds/github.py

```diff
@@ -79,16 +80,11 @@ def is_pre_mds_040(ref) -> bool:
         bool
     """
 
-    str_split = str(ref).split('.')
-
-
-    if not len(str_split) == 3:
-        # not a known mds version
+    try:
+        if ref < Version('0.4.0'):
```

Comment:
I think `ref` is assumed to be a string here, should probably wrap in a `Version()` ctor for comparison.

Reply:
Thanks @thekaveman , I just refactored the code in this function and force-pushed. May you please take a second look when convenient?

Thanks!